layouts/partials/widgets/contact.html

```diff
@@ -76,9 +76,17 @@
 
     {{ if and $.Site.Params.latitude $.Site.Params.longitude }}
     <div class="hidden">
+      {{ if $.Site.Params.google_maps_api_key }}
```

Comment:
Update If/Else statement to check value of `maps_api_key`, similar to above review comments